src/main/kotlin/rhmodding/bread/editor/AnimationsTab.kt

```diff
@@ -170,6 +174,62 @@ open class AnimationsTab<F : IDataModel>(editor: Editor<F>) : EditorSubTab<F>(ed
                     }
                 }
             }
+            children += HBox().apply {
+                styleClass += "hbox"
+                alignment = Pos.CENTER_LEFT
+                children += Button("Export Step").apply {
+                    tooltip = Tooltip("Exports current frame of animation to a PNG file.")
+                    disableProperty().bind(disableStepControls)
+
+                    setOnAction {
+                        val ani = currentAnimation
+                        val curStep = aniStepSpinner.value
+
+                        val padCount: Int = ani.steps.size.toString().length.coerceAtLeast(3)
+                        val stepNum: String = curStep.toString().padStart(padCount, padChar = '0')
+
+                        val fileChooser = FileChooser()
+                        fileChooser.title = "Export the current step of this animation as a PNG image"
+                        fileChooser.extensionFilters.add(FileChooser.ExtensionFilter("PNG", "*.png"))
+                        fileChooser.initialDirectory = editor.dataFile.parentFile
+                        fileChooser.initialFileName = "${getAnimationNameForGifExport()}.$stepNum.png"
+
+                        val file = fileChooser.showSaveDialog(null)
+                        if (file != null) {
+                            val writableImage = writeTransparentSnapshot(ani.steps[curStep])
+
+                            ImageIO.write(SwingFXUtils.fromFXImage(writableImage, null), "png", file)
+                        }
+                        editor.repaintCanvas()
+                    }
+                }
+                children += Button("Export All Steps").apply {
+                    tooltip = Tooltip("Exports each frame of animation to PNG, not including delays.")
```

Comment:
"Exports each step of the animation as a PNG file. The current panning and zoom level will be used."